android/app/src/main/java/com/zecwalletmobile/RPCModule.kt

```diff
@@ -34,7 +34,7 @@ class RPCModule internal constructor(private val reactContext: ReactApplicationC
     @ReactMethod
     fun walletExists(promise: Promise) {
         // Check if a wallet already exists
-        val file = File(MainApplication.getAppContext()?.filesDir, "wallet.dat")
+        val file = File(MainApplication.getAppContext()?.filesDir, "Zingo.dat")
```

Comment:
I believe that this is one of the places where the string `wallet` is appropriate.   We should match the filename that is written by `zingo-cli`.   @dannasessha can you tell us what that filename is?

Reply:
I think It is only something between the App installed (in some Android device) and the internal storage of that device. The name of this file is something you can change if you want to, all above now before the first version. It doesn't matter to me, if you prefer the old one, perfect.